cg/meta/workflow/mutant.py

```diff
@@ -80,7 +80,7 @@ def get_sample_parameters(self, sample_obj: models.Sample) -> MutantSampleConfig
             case_ID=sample_obj.links[0].family.internal_id,
             Customer_ID_sample=sample_obj.name,
             customer_id=sample_obj.customer.internal_id,
-            sequencing_qc_pass=self.check_sample_read_count_above_threshold(sample_obj=sample_obj),
+            sequencing_qc_pass=sample_obj.sequencing_qc,
             CG_ID_project=sample_obj.links[0].family.name,
             Customer_ID_project=sample_obj.ticket_number,
```

Comment:
Could you please update the variable names so that they follow the rest of the code? and add type hints

Reply:
cant do this on kwargs, but you can see the types when hovering in the pycharm